doc/sidebars.ts

```diff
@@ -28,6 +28,7 @@ const sidebars: SidebarsConfig = {
       label: 'Guides',
       items: [
         { type: 'doc', id: 'guides/README', label: 'Introduction' },
+        { type: 'doc', id: 'guides/reformatter', label: 'Usage as Emacs reformatter' },
```

Comment:
Please align the label with title of the page so they're an exact match:

label: Usage as Emacs reformatter
title: Using `dunolint` as an Emacs reformatter

Reply:
Done, modulo the fact that markdown is not supported in labels so I stripped the monospaced font on dunolint.